proto/internalapi/central/compliance_operator.proto

```diff
@@ -125,28 +137,52 @@ message ComplianceOperatorCheckResultV2 {
         INCONSISTENT   = 7;
     }
 
-    enum RuleSeverity {
-        UNSET_RULE_SEVERITY = 0;
-        UNKNOWN_RULE_SEVERITY = 1;
-        INFO_RULE_SEVERITY = 2;
-        LOW_RULE_SEVERITY = 3;
-        MEDIUM_RULE_SEVERITY = 4;
-        HIGH_RULE_SEVERITY = 5;
+    string                         id           = 1;
+    string                         check_id     = 2;
+    string                         check_name   = 3;
+    string                         cluster_id   = 4;
+    CheckStatus                    status       = 5;
+    ComplianceOperatorRuleSeverity severity     = 6;
+    string                         description  = 7;
+    string                         instructions = 8;
+    map<string, string>            labels       = 9;
+    map<string, string>            annotations  = 10;
+    google.protobuf.Timestamp      created_time = 11;
+    string                         standard     = 12;
+    string                         control      = 13;
+    string                         scan_name    = 14;
+    string                         suite_name   = 15;
+}
+
+// ComplianceOperatorProfileV2 is a message from Sensor (to Central) representing a compliance check profile.
+// Next tag: 10.
+message ComplianceOperatorProfileV2 {
+    string              id              = 1;
+    string              profile_id      = 2;
+    string              name            = 3;
```

Comment:
since title field was included in the Rule below, the profile CRD also has a title, besides the name in the ObjectMeta wondering if we need this